packages/proto-loader/bin/proto-loader-gen-types.ts

```diff
@@ -916,6 +920,8 @@ async function runScript() {
       outputTemplate: 'Template for mapping output or "restricted" type names',
       inputBranded: 'Output property for branded type for  "permissive" types with fullName of the Message as its value',
       outputBranded: 'Output property for branded type for  "restricted" types with fullName of the Message as its value',
+      targetFileExtension: 'File extension for generated files. Defaults to .ts',
+      importFileExtension: 'File extension for import specifiers in generated code. Defaults to none (omitted)'
```

Comment:
For consistency with other description messages, don't state the default values. As the README changes show the defaults are included automatically. And update the README again with that change.